sites.json

```diff
@@ -9,6 +9,11 @@
     "site_title_prefix": "BGR",
     "theme_repo": "git@bitbucket.org:penskemediacorp/bgr.git"
   },
+  "billboard.com": {
```

Comment:
Each of these sites needs the `"theme_dir_uses_vip": true` argument in here too, otherwise the next run of `node generate-config.js` will remove those settings from `config.yml`.